client/api/service.py

```diff
@@ -0,0 +1,41 @@
+from FileReader.DataInstance import DataInstance
+# from input.Model import Model
+
+from ipaddress import ip_address
+from urllib.parse import urlparse
+
+from client.api.request import Request
+from client.api.connection_handler import ConnectionHandler
+
+
+class Service:
+
+    def __init__(self, address):
```

Comment:
Jeśli to będzie prostsze, to możemy założyć, że potrzebujemy oddzielnie `ip` i `port`, wtedy też mamy pewność, że muszą podać pewien port, i nie musimy parsować tego